requirements.txt

```diff
@@ -1,3 +1,3 @@
 pymongo==2.7.1
 wheel==0.23.0
-
+numpy==1.9
```

Comment:
I could write the code myself, or copy a solution off StackOverflow:
http://stackoverflow.com/questions/1181919/python-base-36-encoding
But I prefer to use the numpy solution because it's extremely short and no doubt very well matured and bug-free.

All the processing of requirements is done offline, and hard drive space is cheap, so I'm not worried about the size of numpy.